api/v1beta1/machine_types.go

```diff
@@ -230,10 +230,32 @@ type MachineStatus struct {
 	// Conditions defines current service state of the Machine.
 	// +optional
 	Conditions Conditions `json:"conditions,omitempty"`
+
+	// Deletion contains information relating to removal of the Machine.
+	// Only present when the Machine has a deletionTimestamp and is being removed from the cluster.
+	// +optional
```

Comment:
Should this be MachineDeletionStatus?

(the status proposal was proposing MachineInitializationStatus, so MachineStatusDeletion sounds inconsistent)

Reply:
No strong opinion though, we should just end up with something consistent. @JoelSpeed WDYT?

Reply:
Yes agreed, the type should be `MachineDeletionStatus`, status is typically the last word in these structs

From a serialised point of view, `machineDeletion` might be best?

Reply:
I would think we don't need machine for the serialized version as this will be Machine.status.deletion?